hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -36,6 +36,10 @@ private ReconConfigKeys() {
       "ozone.recon.datanode.address";
   public static final String OZONE_RECON_ADDRESS_KEY =
       "ozone.recon.address";
+  public static final String OZONE_SOLR_ADDRESS_KEY =
+      "ozone.solr.address";
```

Comment:
ozone.solr.address,  can we call it "solr.address" instead？ “ozone.solr.address" looks like ozone is providing solr service.